packages/ember-testing/lib/helpers/fill_in.js

```diff
@@ -11,10 +10,10 @@ export default function fillIn(app, selector, contextOrText, text) {
   $el = app.testHelpers.findWithAssert(selector, context);
   el = $el[0];
   focus(el);
-  run(() => {
```

Comment:
we should cleanup any other test helpers that are nesting run loops in the wait one.